pkg/operator/ceph/csi/csi.go

```diff
@@ -37,7 +37,10 @@ func ValidateAndStartDrivers(clientset kubernetes.Interface, namespace, rookImag
 		logger.Errorf("failed to start Ceph csi drivers. %v", err)
 		return
 	}
-	logger.Infof("successfully started Ceph CSI driver(s)")
+	if err := stopDrivers(namespace, clientset, serverVersion); err != nil {
```

Comment:
How about renaming the func on line 30: `ValidateAndConfigureDrivers()`

Reply:
yeah, sounds more relatable.  Changes done.

Reply:
I think we don't need to `validateCSIVersion` if we are stopping the CSI driver. 

we need to change the order
* Check CSI Driver need to be stopped if yes delete it
* VlidateCSIVersion
* StartCSIDrvier

@travisn any though on this order

Reply:
We really only need to validate the CSI version when we are starting the CSI driver. So maybe we should move ValidateCSIVersion inside the startDrivers() method. But that could be a separate PR.